zulipterminal/helper.py

```diff
@@ -53,7 +75,7 @@
     edited_messages=set(),
     topics=defaultdict(list),
     search=set(),
-    messages=defaultdict(dict),
+    messages=defaultdict(lambda: Message()),
```

Comment:
Good catch of the `mypy` bug! It would be useful to put a comment here referring to why it is necessary to use the `lambda`, referring to eg. the `mypy` issue.